src/pages/ImportData.js

```diff
@@ -135,16 +136,19 @@ const ImportData = () => {
         Utils.switchCase(mode,
           ['existing', () => {
             return h(Fragment, [
-              div({ style: styles.title }, ['Start with an existing workspace']),
-              div({ style: { fontWeight: 600, marginBottom: '0.25rem' } }, ['Select one of your workspaces']),
-              h(WorkspaceSelector, {
-                workspaces: _.filter(ws => {
-                  return Utils.canWrite(ws.accessLevel) &&
-                    (!ad || _.some({ membersGroupName: ad }, ws.workspace.authorizationDomain))
-                }, workspaces),
-                value: selectedWorkspaceId,
-                onChange: setSelectedWorkspaceId
-              }),
+              h2({ style: styles.title }, ['Start with an existing workspace']),
+              h(IdContainer, [id => h(Fragment, [
+                h(FormLabel, { htmlFor: id, style: { marginBottom: '0.25rem' } }, ['Select one of your workspaces']),
+                h(WorkspaceSelector, {
+                  id,
+                  workspaces: _.filter(ws => {
+                    return Utils.canWrite(ws.accessLevel) &&
+                      (!ad || _.some({ membersGroupName: ad }, ws.workspace.authorizationDomain))
```

Comment:
Not your code, but this was an interesting case. I generally want the ability for a variable to describe itself to vary with the physical size of the scope. `x` is preferred when contained within a few lines, since the context is readily visible, whereas something more descriptive is better when the scope is larger. In this case, `ad` may be fine even though the scope is a bit larger because it appears to be a common pattern.